1.1/spec/02-Introduction.adoc

```diff
@@ -42,30 +42,42 @@ The OGC GeoSPARQL standard supports representing and querying geospatial data on
 
 === GeoSPARQL Standard structure
 
-The GeoSPARQL standard comprises multiple parts:
+The GeoSPARQL standard comprises multiple parts, or _profile resources_. The comprehensive listing of them is given not here but in the standard's _profile definition, see http://www.opengis.net/def/geosparql. Here is an overview of the major parts:
```

Comment:
At the moment http://www.opengis.net/def/geosparql does not resolve. Are we certain it will resolve to a profile definition when 1.1 is released?

Reply:
Yes, we are able to ask the OGC Naming Authority for this to resolve.